src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,13 @@
+body,
+h2,
+p,
+ul,
+li,
+span {
+  margin: 0;
+  padding: 0;
+  * {
+    box-sizing: border-box;
+    font-family: "Roboto", sans-serif;
+  }
```

Comment:
move font-family property to html tag
and styles for * selector should be separated from other selectors
```suggestion
body,
h2,
p,
ul,
li,
span {
  margin: 0;
  padding: 0;
}

 * {
    box-sizing: border-box;
  }
```